### PR TITLE
Skw payment type

### PR DESCRIPTION
**Pull request by streatorward**
## Link to Ticket
Closes # [PatmentType](https://github.com/NSS-Wacky-Eels/BangazonAPI/issues/1)

## Description of Proposed Changes
Adding PaymentTypes Model and Controller with all functionality. 
- 

## Steps to Test
Need to test in Postman:
Get all
Get Single
Post 
Put
Delete

```sh
git fetch --all
git checkout skwPaymentType

```

## Impacted Areas in Application
New Model and Controller for PaymentTypes will be added to respective Folders.

*

### CODE REVIEW

**Review by MissPeperr — APPROVED** [review]
All CRUD operations work!

**Review by taylorlgulley — COMMENTED** [review]
All the CRUD functions works.
Everything looks good.
👍

**Review by taylorlgulley — APPROVED** [review]
👍

File: Bangazon/Models/PaymentType.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+
+namespace Bangazon.Models
+{
+    public class PaymentType
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string AcctNumber { get; set; }
```
Comment (askingalot) [inline Bangazon/Models/PaymentType.cs]: The data type for `AcctNumber` is an `int` in the database, so should be one in the model.

File: Bangazon/Controllers/PaymentTypesController.cs [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+            get
+            {
+                return new SqlConnection(_config.GetConnectionString("DefaultConnection"));
+            }
+        }
+
+
+        // GET api/paymenttypes
+        [HttpGet]
+        public async Task<IActionResult> Get(string q)
```
Comment (askingalot) [inline Bangazon/Controllers/PaymentTypesController.cs]: The ticket does mention anything about a `q` parameter. It is not needed.

**Review by askingalot — CHANGES_REQUESTED** [review]
This is very close, but there are a couple off changes. See my other comments.

File: Bangazon/Models/PaymentType.cs (hunk 1 above)
Comment (streatorward) [reply Bangazon/Models/PaymentType.cs]: I have made the proper changes to this

File: Bangazon/Controllers/PaymentTypesController.cs (hunk 2 above)
Comment (streatorward) [reply Bangazon/Controllers/PaymentTypesController.cs]: I have made the proper changes to this

**Comment by streatorward** [comment]
I have pushed up again and fixed the 2 changes from the comments above

**Comment by stevebrownlee** [comment]
@streatorward Approved

**Review by stevebrownlee — APPROVED** [review]

**Pull request merged**